src/app/shapes/Line.ts

```diff
@@ -6,33 +6,28 @@ export class Line extends Shape {
   private strokeWidth: number = 2;
   private static DEFAULT_COLOR: string = "black";
 
-  private x1: number;
-  private y1: number;
-  private x2: number;
-  private y2: number;
-  
   public constructor(name: string, x1: number, y1: number, x2: number, y2: number) {
```

Comment:
If you use a Konva.Line you won't need these.

Reply:
They would get sent in points

Reply:
This PR removed those..?

Reply:
🤦‍♀️